docs/glossary.md

```diff
@@ -146,7 +146,12 @@ and [commits](#commit).
 
 ## Remote
 
-TODO
+A remote is a reference to a copy of your repository. The most common case is
+that the remote is hosted on the internet or another network. Remotes are
```

Comment:
nit: also worth it to mention local remotes since `jj git add remote <name> <local-repo>` works